src/Entity/User.php

```diff
@@ -50,6 +50,11 @@ class User implements UserInterface
      */
     private $homepage = "";
 
+    /**
+     * @ORM\Column(type="string", length=255)
```

Comment:
```suggestion
     * @ORM\Column(type="string", length=255,  nullable=true)
```

Kai keičiame sistemą – reikia leisti nenurodyti kažkurio laukelio.
Kitu atveju reikėtų su laukelio pridėjimu praleisti ir visų senų naudotojų duomenų migraciją (transformavimą).

![image](https://user-images.githubusercontent.com/1453957/70869108-17024d80-1f90-11ea-8b2e-181ae421d750.png)